### ISSUE TITLE
Crash in 9.1

### ISSUE THREAD

**Issue by weedmapsmike** [Instabug/Instabug-iOS#362]
### Steps to Reproduce the Problem
In our app we are logging a user in and that triggers the crash. it is unclear which methods from the sdk is triggering the crash.

### Expected Behavior
The app should not crash.

### Actual Behavior
The app crashes.

### Instabug integration code
*Make sure to remove your app token.*

### SDK Version
9.1

### iOS Version
13.4

### Device Model
iPhone 11 Pro Max

### [Optional] Project That Reproduces the Issue
Don't have a project that repros the issue, but this is the crash call stack:

```
Thread 1 Queue : com.apple.main-thread (serial)
#0	0x000000010ff7e711 in dispatch_group_leave.cold.1 ()
#1	0x000000010ff4cafa in dispatch_group_leave ()
#2	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#3	0x000000010ff4be8e in _dispatch_client_callout ()
#4	0x000000010ff59d97 in _dispatch_main_queue_callback_4CF ()
#5	0x00007fff23da1849 in __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ ()
#6	0x00007fff23d9c399 in __CFRunLoopRun ()
#7	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#8	0x00007fff38b5ac1a in GSEventRunModal ()
#9	0x00007fff48c19220 in UIApplicationMain ()
#10	0x000000010993c08f in main at path_to_project/Sources/Bootstrap/main.swift:7
#11	0x00007fff519b910d in start ()
#12	0x00007fff519b910d in start ()
Enqueued from NSManagedObjectContext 0x6000029b2bc0 (Thread 18) Queue : NSManagedObjectContext 0x6000029b2bc0 (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x000000010d7e3389 in __76-[IBGManagedObjectContextController saveChangesToDiskWithCompeltionHandler:]_block_invoke at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:97
#2	0x00007fff23a034c9 in developerSubmittedBlockToNSManagedObjectContextPerform ()
#3	0x000000010ff4be8e in _dispatch_client_callout ()
#4	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#5	0x000000010ff5328f in _dispatch_lane_invoke ()
#6	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#7	0x00007fff51b9ea3d in _pthread_wqthread ()
#8	0x00007fff51b9db77 in start_wqthread ()
Enqueued from com.apple.main-thread (Thread 1) Queue : com.apple.main-thread (serial)
#0	0x000000010ff4fa61 in dispatch_async_f ()
#1	0x000000010d7e30d2 in -[IBGManagedObjectContextController saveChangesToDiskWithCompeltionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:82
#2	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#3	0x000000010ff4be8e in _dispatch_client_callout ()
#4	0x000000010ff59d97 in _dispatch_main_queue_callback_4CF ()
#5	0x00007fff23da1849 in __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ ()
#6	0x00007fff23d9c399 in __CFRunLoopRun ()
#7	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#8	0x00007fff38b5ac1a in GSEventRunModal ()
#9	0x00007fff48c19220 in UIApplicationMain ()
#10	0x000000010993c08f in main at path_to_project/Sources/Bootstrap/main.swift:7
#11	0x00007fff519b910d in start ()
Enqueued from NSManagedObjectContext 0x6000029b2d80 (Thread 17) Queue : NSManagedObjectContext 0x6000029b2d80 (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x000000010d7e2f41 in -[IBGManagedObjectContextController saveWithCompletionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:73
#2	0x000000010d7e1427 in -[IBGCoreDataLayer saveWithCompletionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGCoreDataLayer.m:50
#3	0x000000010d7e2097 in __44-[IBGDALService writeASync:completionBlock:]_block_invoke at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGDALService.m:121
#4	0x00007fff23a034c9 in developerSubmittedBlockToNSManagedObjectContextPerform ()
#5	0x000000010ff4be8e in _dispatch_client_callout ()
#6	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#7	0x000000010ff5328f in _dispatch_lane_invoke ()
#8	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#9	0x00007fff51b9ea3d in _pthread_wqthread ()
#10	0x00007fff51b9db77 in start_wqthread ()
Thread 2 Queue : DefaultEventDispatcherQueue (serial)
#0	0x00007fff51af255e in __ulock_wait ()
#1	0x000000010ff4c7f2 in _dlock_wait ()
#2	0x000000010ff4ca80 in _dispatch_group_wait_slow ()
#3	0x000000010e578cc8 in closure #1 in DefaultEventDispatcher.flushEvents() at /Users/mshabsin/Projects/weedmaps-bitrise/weedmaps-ios/Pods/OptimizelySwiftSDK/Sources/Customization/DefaultEventDispatcher.swift:176
#4	0x000000010e525110 in thunk for @escaping @callee_guaranteed () -> () ()
#5	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#6	0x000000010ff4be8e in _dispatch_client_callout ()
#7	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#8	0x000000010ff5328f in _dispatch_lane_invoke ()
#9	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#10	0x00007fff51b9ea3d in _pthread_wqthread ()
#11	0x00007fff51b9db77 in start_wqthread ()
Thread 3 Queue : com.google.fira.worker (serial)
#0	0x00007fff51af2336 in stat$INODE64 ()
#1	0x00007fff50c43bef in unixSync ()
#2	0x00007fff50c54342 in syncJournal ()
#3	0x00007fff50c1b6be in sqlite3PagerCommitPhaseOne ()
#4	0x00007fff50c1bae3 in sqlite3BtreeCommitPhaseOne ()
#5	0x00007fff50c6029b in sqlite3VdbeHalt ()
#6	0x00007fff50c75a33 in sqlite3VdbeExec ()
#7	0x00007fff50c1766e in sqlite3_step ()
#8	0x00007fff50c25335 in sqlite3_exec ()
#9	0x0000000109be4f17 in -[APMSqliteStore endTransaction] ()
#10	0x0000000109be4d8d in -[APMSqliteStore performTransactionWithError:block:] ()
#11	0x0000000109b9dd69 in -[APMDatabase performTransaction:] ()
#12	0x0000000109bb8422 in -[APMMeasurement writeEventOnWorkerQueue:] ()
#13	0x0000000109bb7834 in -[APMMeasurement handleEventOnWorkerQueue:] ()
#14	0x0000000109bb6aff in __27-[APMMeasurement logEvent:]_block_invoke ()
#15	0x0000000109bcd937 in __51-[APMScheduler scheduleOnWorkerQueueBlockID:block:]_block_invoke ()
#16	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#17	0x000000010ff4be8e in _dispatch_client_callout ()
#18	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#19	0x000000010ff5328f in _dispatch_lane_invoke ()
#20	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#21	0x00007fff51b9ea3d in _pthread_wqthread ()
#22	0x00007fff51b9db77 in start_wqthread ()
Thread 4#0	0x00007fff51af24ce in __workq_kernreturn ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 5 Queue : Anonymous
#0	0x00007fff51af0e36 in semaphore_wait_trap ()
#1	0x000000010ff4c489 in _dispatch_sema4_wait ()
#2	0x000000010ff4c988 in _dispatch_semaphore_wait_slow ()
#3	0x000000010935be3a in OnboardingStartupOperation.startupOperationMain() at path_to_project/Sources/Bootstrap/StartupOperations.swift:105
#4	0x00000001098e59b3 in StartupOperation.main() at path_to_project/Sources/Bootstrap/StartupManager.swift:37
#5	0x00000001098e5c7b in @objc StartupOperation.main() ()
#6	0x00007fff25917c9c in __NSOPERATION_IS_INVOKING_MAIN__ ()
#7	0x00007fff25913fae in -[NSOperation start] ()
#8	0x00007fff259185e0 in __NSOPERATIONQUEUE_IS_STARTING_AN_OPERATION__ ()
#9	0x00007fff25918100 in __NSOQSchedule_f ()
#10	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#11	0x000000010ff4be8e in _dispatch_client_callout ()
#12	0x000000010ff4e7a3 in _dispatch_continuation_pop ()
#13	0x000000010ff4dbbb in _dispatch_async_redirect_invoke ()
#14	0x000000010ff5d399 in _dispatch_root_queue_drain ()
#15	0x000000010ff5dca6 in _dispatch_worker_thread2 ()
#16	0x00007fff51b9e9f7 in _pthread_wqthread ()
#17	0x00007fff51b9db77 in start_wqthread ()
Enqueued from  (Thread 5) Queue :  (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x00007fff25912163 in __NSOQSchedule ()
#2	0x00007fff25911dd2 in +[__NSOperationInternalObserver _observeValueForKeyPath:ofObject:changeKind:oldValue:newValue:indexes:context:] ()
#3	0x00007fff2591197d in +[__NSOperationInternalObserver _observeValueForKeyPath:ofObject:changeKind:oldValue:newValue:indexes:context:] ()
#4	0x00007fff259141ab in -[NSOperation start] ()
#5	0x00007fff259185e0 in __NSOPERATIONQUEUE_IS_STARTING_AN_OPERATION__ ()
#6	0x00007fff25918100 in __NSOQSchedule_f ()
#7	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#8	0x000000010ff4be8e in _dispatch_client_callout ()
#9	0x000000010ff4e7a3 in _dispatch_continuation_pop ()
#10	0x000000010ff4dbbb in _dispatch_async_redirect_invoke ()
#11	0x000000010ff5d399 in _dispatch_root_queue_drain ()
#12	0x000000010ff5dca6 in _dispatch_worker_thread2 ()
#13	0x00007fff51b9e9f7 in _pthread_wqthread ()
#14	0x00007fff51b9db77 in start_wqthread ()
Enqueued from com.apple.main-thread (Thread 1) Queue : com.apple.main-thread (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x00007fff25912163 in __NSOQSchedule ()
#2	0x00007fff25915f29 in __addOperations ()
#3	0x00000001098e7a06 in static StartupManager.queue(_:beforeMainExitClosure:) at path_to_project/Sources/Bootstrap/StartupManager.swift:108
#4	0x00000001098ad566 in AppDelegate.configureStartupManager() at path_to_project/Sources/Bootstrap/AppDelegate.swift:129
#5	0x00000001098b0c3a in AppDelegate.application(_:didFinishLaunchingWithOptions:) at path_to_project/Sources/Bootstrap/AppDelegate.swift:241
#6	0x00000001098b0d23 in @objc AppDelegate.application(_:didFinishLaunchingWithOptions:) ()
#7	0x00007fff48c0fecc in -[UIApplication _handleDelegateCallbacksWithOptions:isSuspended:restoreState:] ()
#8	0x00007fff48c1186b in -[UIApplication _callInitializationDelegatesWithActions:forCanvas:payload:fromOriginatingProcess:] ()
#9	0x00007fff48c1742d in -[UIApplication _runWithMainScene:transitionContext:completion:] ()
#10	0x00007fff48322a61 in -[_UISceneLifecycleMultiplexer completeApplicationLaunchWithFBSScene:transitionContext:] ()
#11	0x00007fff4882d065 in _UIScenePerformActionsWithLifecycleActionMask ()
#12	0x00007fff48323573 in __101-[_UISceneLifecycleMultiplexer _evalTransitionToSettings:fromSettings:forceExit:withTransitionStore:]_block_invoke ()
#13	0x00007fff48322f82 in -[_UISceneLifecycleMultiplexer _performBlock:withApplicationOfDeactivationReasons:fromReasons:] ()
#14	0x00007fff483233a0 in -[_UISceneLifecycleMultiplexer _evalTransitionToSettings:fromSettings:forceExit:withTransitionStore:] ()
#15	0x00007fff48322c35 in -[_UISceneLifecycleMultiplexer uiScene:transitionedFromState:withTransitionContext:] ()
#16	0x00007fff48327226 in __186-[_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction _performActionsForUIScene:withUpdatedFBSScene:settingsDiff:fromSettings:transitionContext:lifecycleActionType:]_block_invoke_2 ()
#17	0x00007fff48743415 in +[BSAnimationSettings(UIKit) tryAnimatingWithSettings:actions:completion:] ()
#18	0x00007fff4884b7c2 in _UISceneSettingsDiffActionPerformChangesWithTransitionContext ()
#19	0x00007fff48326f41 in __186-[_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction _performActionsForUIScene:withUpdatedFBSScene:settingsDiff:fromSettings:transitionContext:lifecycleActionType:]_block_invoke ()
#20	0x00007fff4884b6c5 in _UISceneSettingsDiffActionPerformActionsWithDelayForTransitionContext ()
#21	0x00007fff48326daf in -[_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction _performActionsForUIScene:withUpdatedFBSScene:settingsDiff:fromSettings:transitionContext:lifecycleActionType:] ()
#22	0x00007fff4817a968 in __64-[UIScene scene:didUpdateWithDiff:transitionContext:completion:]_block_invoke ()
#23	0x00007fff48179527 in -[UIScene _emitSceneSettingsUpdateResponseForCompletion:afterSceneUpdateWork:] ()
#24	0x00007fff4817a692 in -[UIScene scene:didUpdateWithDiff:transitionContext:completion:] ()
#25	0x00007fff48c15975 in -[UIApplication workspace:didCreateScene:withTransitionContext:completion:] ()
#26	0x00007fff4876a160 in -[UIApplicationSceneClientAgent scene:didInitializeWithEvent:completion:] ()
#27	0x00007fff36c6091e in -[FBSSceneImpl _callOutQueue_agent_didCreateWithTransitionContext:completion:] ()
#28	0x00007fff36c869b1 in __86-[FBSWorkspaceScenesClient sceneID:createWithParameters:transitionContext:completion:]_block_invoke.154 ()
#29	0x00007fff36c6b347 in -[FBSWorkspace _calloutQueue_executeCalloutFromSource:withBlock:] ()
#30	0x00007fff36c86642 in __86-[FBSWorkspaceScenesClient sceneID:createWithParameters:transitionContext:completion:]_block_invoke ()
#31	0x000000010ff4be8e in _dispatch_client_callout ()
#32	0x000000010ff4eda2 in _dispatch_block_invoke_direct ()
#33	0x00007fff36cac2d9 in __FBSSERIALQUEUE_IS_CALLING_OUT_TO_A_BLOCK__ ()
#34	0x00007fff36cabfc7 in -[FBSSerialQueue _queue_performNextIfPossible] ()
#35	0x00007fff36cac4d6 in -[FBSSerialQueue _performNextFromRunLoopSource] ()
#36	0x00007fff23da1c71 in __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ ()
#37	0x00007fff23da1b9c in __CFRunLoopDoSource0 ()
#38	0x00007fff23da1374 in __CFRunLoopDoSources0 ()
#39	0x00007fff23d9bf6e in __CFRunLoopRun ()
#40	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#41	0x00007fff38b5ac1a in GSEventRunModal ()
#42	0x00007fff48c19220 in UIApplicationMain ()
#43	0x000000010993c08f in main at path_to_project/Sources/Bootstrap/main.swift:7
#44	0x00007fff519b910d in start ()
com.apple.uikit.eventfetch-thread (6)#0	0x00007fff51af0dfa in mach_msg_trap ()
#1	0x00007fff51af1170 in mach_msg ()
#2	0x00007fff23da14d5 in __CFRunLoopServiceMachPort ()
#3	0x00007fff23d9c107 in __CFRunLoopRun ()
#4	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#5	0x00007fff2592d831 in -[NSRunLoop(NSRunLoop) runMode:beforeDate:] ()
#6	0x00007fff2592daa0 in -[NSRunLoop(NSRunLoop) runUntilDate:] ()
#7	0x00007fff48cc6057 in -[UIEventFetcher threadMain] ()
#8	0x00007fff259435ab in __NSThread__start__ ()
#9	0x00007fff51ba2109 in _pthread_start ()
#10	0x00007fff51b9db8b in thread_start ()
Thread 7 Queue : com.apple.network.connections (serial)
#0	0x00007fff51af0dfa in mach_msg_trap ()
#1	0x00007fff51af1170 in mach_msg ()
#2	0x00007fff51beb14f in _xpc_send_serializer ()
#3	0x00007fff51beafb2 in _xpc_pipe_simpleroutine ()
#4	0x00007fff51beb317 in xpc_pipe_simpleroutine ()
#5	0x00007fff51bb8247 in ___os_activity_stream_reflect_block_invoke_2 ()
#6	0x000000010ff4be8e in _dispatch_client_callout ()
#7	0x000000010ff4eda2 in _dispatch_block_invoke_direct ()
#8	0x000000010ff4ec6b in dispatch_block_perform ()
#9	0x00007fff51bb7cd5 in _os_activity_stream_reflect ()
#10	0x00007fff51bc3433 in _os_log_impl_stream ()
#11	0x00007fff51bc29f7 in _os_log_impl_flatten_and_send ()
#12	0x00007fff51bc408e in os_log_pack_send ()
#13	0x00007fff5087b1fa in __nw_association_schedule_deactivation_block_invoke ()
#14	0x000000010ff4be8e in _dispatch_client_callout ()
#15	0x000000010ff4e7a3 in _dispatch_continuation_pop ()
#16	0x000000010ff61b2c in _dispatch_source_invoke ()
#17	0x000000010ff5478c in _dispatch_workloop_invoke ()
#18	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#19	0x00007fff51b9ea3d in _pthread_wqthread ()
#20	0x00007fff51b9db77 in start_wqthread ()
Thread 8com.apple.NSURLConnectionLoader (9)#0	0x00007fff51af0dfa in mach_msg_trap ()
#1	0x00007fff51af1170 in mach_msg ()
#2	0x00007fff23da14d5 in __CFRunLoopServiceMachPort ()
#3	0x00007fff23d9c107 in __CFRunLoopRun ()
#4	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#5	0x00007fff2315a073 in ___lldb_unnamed_symbol11344$$CFNetwork ()
#6	0x00007fff259435ab in __NSThread__start__ ()
#7	0x00007fff51ba2109 in _pthread_start ()
#8	0x00007fff51b9db8b in thread_start ()
Thread 10 Queue : AXBundleLoadQueue (serial)
#0	0x000000011603efc6 in getattrlist ()
#1	0x00007fff51a35340 in _readdir_unlocked ()
#2	0x00007fff51a35410 in readdir ()
#3	0x00007fff23d51103 in _CFIterateDirectory ()
#4	0x00007fff23d20b5c in _CFBundleGetBundleVersionForURL ()
#5	0x00007fff23d1aab6 in _CFBundleCreate ()
#6	0x00007fff2588a1a3 in -[NSBundle _cfBundle] ()
#7	0x00007fff2588afd2 in -[NSBundle loadAndReturnError:] ()
#8	0x00007fff48029a2c in -[NSBundleAccessibility loadAndReturnError:] ()
#9	0x00007fff2e93b055 in __50-[AXCodeItem loadWithStrategy:onQueue:completion:]_block_invoke_2 ()
#10	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#11	0x000000010ff4be8e in _dispatch_client_callout ()
#12	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#13	0x000000010ff5328f in _dispatch_lane_invoke ()
#14	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#15	0x00007fff51b9ea3d in _pthread_wqthread ()
#16	0x00007fff51b9db77 in start_wqthread ()
Thread 11 Queue : com.google.perf.FPREventsQueue (serial)
#0	0x00007fff51af25ee in __ulock_wake ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 12#0	0x00007fff51b9db68 in start_wqthread ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
GAIThread (13)#0	0x00007fff51af0dfa in mach_msg_trap ()
#1	0x00007fff51af1170 in mach_msg ()
#2	0x00007fff23da14d5 in __CFRunLoopServiceMachPort ()
#3	0x00007fff23d9c107 in __CFRunLoopRun ()
#4	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#5	0x00007fff2592d831 in -[NSRunLoop(NSRunLoop) runMode:beforeDate:] ()
#6	0x00007fff2592da45 in -[NSRunLoop(NSRunLoop) run] ()
#7	0x0000000109a3c698 in +[GAI threadMain:] ()
#8	0x00007fff259435ab in __NSThread__start__ ()
#9	0x00007fff51ba2109 in _pthread_start ()
#10	0x00007fff51b9db8b in thread_start ()
Thread 14#0	0x00007fff51af24ce in __workq_kernreturn ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 15#0	0x00007fff51b9db68 in start_wqthread ()
#1	0x0000000109a70c3f in CLSFileLoopWithWriteBlock ()
#2	0x0000000109a70e5a in CLSFileWriteToFileDescriptorOrBuffer ()
#3	0x0000000109a7118e in CLSFileWriteStringWithSuffix ()
#4	0x0000000109a71258 in CLSFileWriteHashEntryString ()
#5	0x0000000109a58922 in __CLSBinaryImageChanged_block_invoke ()
#6	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#7	0x000000010ff4be8e in _dispatch_client_callout ()
#8	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#9	0x000000010ff5328f in _dispatch_lane_invoke ()
#10	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#11	0x00007fff51b9ea3d in _pthread_wqthread ()
#12	0x00007fff51b9db77 in start_wqthread ()
Enqueued from com.apple.main-thread (Thread 1) Queue : com.apple.main-thread (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x0000000109a58853 in CLSBinaryImageChanged ()
#2	0x000000010bf1b65f in dyld::notifyBatchPartial(dyld_image_states, bool, char const* (*)(dyld_image_states, unsigned int, dyld_image_info const*), bool, bool) ()
#3	0x000000010bf244a9 in ImageLoader::link(ImageLoader::LinkContext const&, bool, bool, bool, ImageLoader::RPathChain const&, char const*) ()
#4	0x000000010bf1bdf3 in dyld::link(ImageLoader*, bool, bool, ImageLoader::RPathChain const&, unsigned int) ()
#5	0x000000010bf204d4 in dlopen_internal ()
#6	0x00007fff519bce48 in dlopen ()
#7	0x00000001121dd4b4 in 0x1121dd4b4 ()
#8	0x00000001121dd68d in 0x1121dd68d ()
#9	0x000000010993bf90 in outlined destroy of ProductsResponse? ()
#10	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#11	0x000000010ff4be8e in _dispatch_client_callout ()
#12	0x000000010ff59d97 in _dispatch_main_queue_callback_4CF ()
#13	0x00007fff23da1849 in __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ ()
#14	0x00007fff23d9c399 in __CFRunLoopRun ()
#15	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#16	0x00007fff38b5ac1a in GSEventRunModal ()
#17	0x00007fff48c19220 in UIApplicationMain ()
#18	0x000000010993c08f in main at path_to_project/Sources/Bootstrap/main.swift:7
#19	0x00007fff519b910d in start ()
Enqueued from NSManagedObjectContext 0x6000029b2bc0 (Thread 18) Queue : NSManagedObjectContext 0x6000029b2bc0 (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x000000010d7e3389 in __76-[IBGManagedObjectContextController saveChangesToDiskWithCompeltionHandler:]_block_invoke at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:97
#2	0x00007fff23a034c9 in developerSubmittedBlockToNSManagedObjectContextPerform ()
#3	0x000000010ff4be8e in _dispatch_client_callout ()
#4	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#5	0x000000010ff5328f in _dispatch_lane_invoke ()
#6	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#7	0x00007fff51b9ea3d in _pthread_wqthread ()
#8	0x00007fff51b9db77 in start_wqthread ()
Enqueued from com.apple.main-thread (Thread 1) Queue : com.apple.main-thread (serial)
#0	0x000000010ff4fa61 in dispatch_async_f ()
#1	0x000000010d7e30d2 in -[IBGManagedObjectContextController saveChangesToDiskWithCompeltionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:82
#2	0x000000010ff4af11 in _dispatch_call_block_and_release ()
#3	0x000000010ff4be8e in _dispatch_client_callout ()
#4	0x000000010ff59d97 in _dispatch_main_queue_callback_4CF ()
#5	0x00007fff23da1849 in __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ ()
#6	0x00007fff23d9c399 in __CFRunLoopRun ()
#7	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#8	0x00007fff38b5ac1a in GSEventRunModal ()
#9	0x00007fff48c19220 in UIApplicationMain ()
#10	0x000000010993c08f in main at path_to_project/Sources/Bootstrap/main.swift:7
#11	0x00007fff519b910d in start ()
Enqueued from NSManagedObjectContext 0x6000029b2d80 (Thread 17) Queue : NSManagedObjectContext 0x6000029b2d80 (serial)
#0	0x000000010ff4fd46 in dispatch_async ()
#1	0x000000010d7e2f41 in -[IBGManagedObjectContextController saveWithCompletionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGManagedObjectContextController.m:73
#2	0x000000010d7e1427 in -[IBGCoreDataLayer saveWithCompletionHandler:] at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGCoreDataLayer.m:50
#3	0x000000010d7e2097 in __44-[IBGDALService writeASync:completionBlock:]_block_invoke at /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGDALService.m:121
#4	0x00007fff23a034c9 in developerSubmittedBlockToNSManagedObjectContextPerform ()
#5	0x000000010ff4be8e in _dispatch_client_callout ()
#6	0x000000010ff526fd in _dispatch_lane_serial_drain ()
#7	0x000000010ff5328f in _dispatch_lane_invoke ()
#8	0x000000010ff5eb65 in _dispatch_workloop_worker_thread ()
#9	0x00007fff51b9ea3d in _pthread_wqthread ()
#10	0x00007fff51b9db77 in start_wqthread ()
Thread 16 Queue : io.answers.EventQueue (QOS: BACKGROUND) (concurrent)
#0	0x00007fff51af3ee2 in __unlink ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 17#0	0x00007fff51af24ce in __workq_kernreturn ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 18#0	0x00007fff51af24ce in __workq_kernreturn ()
Thread 19#0	0x00007fff51af24ce in __workq_kernreturn ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
com.apple.CFNetwork.CustomProtocols (20)#0	0x00007fff51af0dfa in mach_msg_trap ()
#1	0x00007fff51af1170 in mach_msg ()
#2	0x00007fff23da14d5 in __CFRunLoopServiceMachPort ()
#3	0x00007fff23d9c107 in __CFRunLoopRun ()
#4	0x00007fff23d9b884 in CFRunLoopRunSpecific ()
#5	0x00007fff2315a073 in ___lldb_unnamed_symbol11344$$CFNetwork ()
#6	0x00007fff259435ab in __NSThread__start__ ()
#7	0x00007fff51ba2109 in _pthread_start ()
#8	0x00007fff51b9db8b in thread_start ()
Thread 21Thread 22#0	0x00007fff51af24ce in __workq_kernreturn ()
#1	0x00007fff51b9eaa1 in _pthread_wqthread ()
#2	0x00007fff51b9db77 in start_wqthread ()
Thread 23
```

**Comment by Kmohamed** [comment]
We are sorry about this crash. We are aware of it an It will be released very soon this week. I will update you once it's released. Also, if you wanna a custom build I can do it for you until the release date.

**Comment by weedmapsmike** [comment]
@Kmohamed thanks for the update. do you have an estimate of when the fix will be released this week?

**Comment by Kmohamed** [comment]
It should be Friday or Monday. Will update you once we do the release

**Comment by joeoreilly** [comment]
Hey. We're seeing the same crash in 9.1, right after login, any idea when the fix will be available?

**Comment by Kmohamed** [comment]
This crash is fixed on 9.1.1. Feel free to open it again if you face it again. 🙏
